misk-embedded-sample/embedded/build.gradle

```diff
@@ -0,0 +1,32 @@
+buildscript {
+  dependencies {
+    classpath dep.shadowJarPlugin
+  }
+}
+
+apply plugin: 'com.github.johnrengelman.shadow'
+
+dependencies {
+  implementation project(':misk-embedded-sample:service')
+  // somehow exclude this from the above dependencies
+  // compile project(':misk-embedded-sample:client')
+}
+
+shadowJar {
+  baseName = 'misk.embedded.sample.embedded.EmbeddedSampleService'
+  classifier = ''
+  archiveVersion = ''
+  dependencies {
+    exclude(project(':misk-embedded-sample:client'))
+    // TODO: implicitly exclude transitive runtime deps of client
```

Comment:
Is this a TODO (of some explicit action to take), or an explanation of something implicit in this current configuration?

Reply:
Something we need to implement. Might get hairy because we might have to traverse transitive dependencies.

Reply:
Real todo for us, will clarify comment